packages/react-dom/src/test-utils/ReactTestUtils.js

```diff
@@ -152,9 +156,59 @@ function validateClassInstance(inst, methodName) {
   );
 }
 
-// a stub element, lazily initialized, used by act() when flushing effects
+// a plain dom element, lazily initialized, used by act() when flushing effects
 let actContainerElement = null;
 
+let didWarnAboutActInNodejs = false;
+let didWarnAboutMessageChannel = false;
+
+let enqueueTask;
+try {
+  // assuming we're in node, let's try to get node's
+  // version of setImmediate, bypassing fake timers if any
+  let r = require; // trick packagers not to bundle this stuff.
+  enqueueTask = r('timers').setImmediate;
+} catch (_err) {
+  // we're in a browser
+  // we can't use regular timers because they may still be faked
+  // so we try MessageChannel+postMessage instead
+  enqueueTask = function(callback) {
+    if (didWarnAboutMessageChannel === false) {
+      didWarnAboutMessageChannel = true;
+      warningWithoutStack(
+        typeof MessageChannel !== 'undefined',
```

Comment:
Wrap in a DEV check plz (I know we have a Babel plugin that does this automatically but we try to be explicit and I'd like to get rid of it in the future.)

Reply:
ack fair, missed it.